.github/workflows/preview.yml

```diff
@@ -4,9 +4,9 @@ name: Preview
 on: [pull_request]
 
 jobs:
-  code-engine:
+  preview-setup:
     # Prevents action to be executed by forks and drafts
-    if: github.event.pull_request.draft == false && github.event.pull_request.head.repo.full_name == github.repository
+    if: github.event.action != 'closed' && github.event.pull_request.draft == false && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
@vabarbosa @Tansito what do you think about:

1. Allow draft PR.
    - If not we need to create fake-draft PRs like [#2408](https://github.com/Qiskit/qiskit.org/pull/2408) to have previews.
2. Remove `github.event.pull_request.head.repo.full_name == github.repository`.
    - I can't imagine the use case where someone fork and creates the PR on the fork and not on the original repository.

So we just have:

```
    if: github.event.action != 'closed'
```

Reply:
Answering point by point:
1. I don't see a problem for this. It's something that we decided in the past but without any constraint to be honest.
2. I think it's something that we just clarified @abdonrd today in a call but just for a bit more context for the rest of the team this is necessary due to the problems with forks: https://github.com/Qiskit/qiskit.org/issues/2276

Reply:
i am ok, with enabling previews for drafts. the reason it was initially disabled was because we weren't properly cleaning up after the PR was closed so we ended up running into space limitations in container registry